server/service/IndividualServicesService.js

```diff
@@ -484,4 +521,34 @@ function getAllApplicationList() {
       reject();
     }
   });
-}
\ No newline at end of file
+}
+
+
+var resolveHttpClient = exports.resolveHttpClientLtpUuidFromForwardingName =  function (forwardingName) {
+  return new Promise(async function (resolve, reject) {
+    try{
+    let ForwardConstructName = await ForwardingDomain.getForwardingConstructForTheForwardingNameAsync(forwardingName)
+    if (ForwardConstructName === undefined) {
+    return null;
+    }
+  let LogicalTerminationPointlist;
+  let httpClientUuidList = [];
+  let ForwardConstructUuid = ForwardConstructName[onfAttributes.GLOBAL_CLASS.UUID]
+  let ListofUuid = await ForwardingConstruct.getFcPortListAsync(ForwardConstructUuid)
+ for (let i = 0; i < ListofUuid.length; i++) {
+ let PortDirection = ListofUuid[i][[onfAttributes.FC_PORT.PORT_DIRECTION]]
+if (PortDirection === FcPort.portDirectionEnum.OUTPUT) {
+LogicalTerminationPointlist = ListofUuid[i][onfAttributes.CONTROL_CONSTRUCT.LOGICAL_TERMINATION_POINT]
+let httpClientUuid = await logicalTerminationPoint.getServerLtpListAsync(LogicalTerminationPointlist)
+let tcpClientUuid = await logicalTerminationPoint.getServerLtpListAsync(httpClientUuid[0])
+
+httpClientUuidList.push(httpClientUuid[0],LogicalTerminationPointlist,tcpClientUuid[0]);
+
+}
+ }
+resolve(httpClientUuidList)
+    }catch(error){
+console.log(error)
+    }
+   })
+  }
```

Comment:
Please format the function properly, with aligned brackets and proper whitespaces.

Reply:
done